lib/solaar/ui/window.py

```diff
@@ -538,11 +522,11 @@ def _details_items(device, read_all=False):
 				for fw in list(device.firmware):
 					yield ('  ' + _(str(fw.kind)), (fw.name + ' ' + fw.version).strip())
 			elif device.kind is None or device.online:
-				yield ('  %s' % _("Firmware"), '...')
+				yield (_("  Firmware"), '...')
```

Comment:
Can you undo this change? The space is for formatting purposes, the original translation should be OK.